docs/UserGuide.md

```diff
@@ -45,7 +49,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `add q/DESCRIPTION a/DESCRIPTION`, `DESCRIPTION` is a parameter which can be used as `add q/hi a/hola`.
```

Comment:
let's discuss the format for the `add` command this week

Reply:
we will use e/ and f/ as discussed. this is already changed and merged into master, and thus can be removed from this PR